src/navigator.js

```diff
@@ -406,6 +406,15 @@ $.extend( $.Navigator.prototype, $.EventSource.prototype, $.Viewer.prototype, /*
         return $.Viewer.prototype.addTiledImage.apply(this, [optionsClone]);
     },
 
+    destroy: function() {
+        this.element.getElementsByTagName('div')[0].removeChild(this.displayRegionContainer);
+        this.displayRegionContainer.removeChild(this.displayRegion);
+        this.viewer.removeControl(this.element);
+        this.innerTracker.destroy();
+        this.outerTracker.destroy();
+        this.element = null;
```

Comment:
Presumably we need this too:

```
return $.Viewer.prototype.destroy.apply(this);
```

Once that's in, we probably don't need to clean up the trackers or the element ourselves either.